packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1761,8 +1762,8 @@ export default class Meeting extends StatelessWebexPlugin {
       LOCUSINFO.EVENTS.CONTROLS_MEETING_TRANSCRIBE_UPDATED,
       ({caption, transcribing}) => {
         // @ts-ignore - config coming from registerPlugin
-        if (transcribing && this.transcription && this.config.receiveTranscription) {
-          this.receiveTranscription();
+        if (transcribing && !this.transcription && this.config.receiveTranscription) {
+          this.startTranscription();
```

Comment:
question: doesn't startTranscription call the service to actually set up a transcription? Doesn't the transcription already exist at this point? Is the API fully idempotent such that this is not an issue ?

Reply:
`startTranscription()` api basically creates a new `transcription` object and assigns it on meeting object and also subscribes to incoming transcriptions.

This method is called while [joining the meeting](https://github.com/meta-dipanshu-sharma/webex-js-sdk/blob/guest-transcription-issue-fix/packages/%40webex/plugin-meetings/src/meeting/index.ts#L3975) also if guest wants to receive transcription but it fails because of empty `datachannelUrl` as I mentioned in the PR description. Then, once we have `datachannelUrl` when user is admitted into the meeting, we are calling `startTranscription` method again which gets completed successfully and user starts receiving transcriptions.

Reply:
ah I see. That makes sense, so it doesn't make an API call, it just starts receiving the open data channel, this makes sense.